File: FITCC.UI/admin_fitcc/lib/models/takmicenje.dart
class Takmicenje {
  List<TakmicenjeItem> takmicenja = [];
}

class TakmicenjeItem {
  late int takmicenjeId;
  late String? naziv;
  late String? slogan;
  late DateTime? pocetak;
  late DateTime? kraj;
  late int? godina;
  late int? brojDana;
  late String? slika;

  TakmicenjeItem(this.takmicenjeId, this.naziv, this.slika, this.pocetak,
      this.kraj, this.godina, this.brojDana, this.slogan);

  factory TakmicenjeItem.fromJson(Map<String, dynamic> map) {
    return TakmicenjeItem(
        map['takmicenjeId'],
        map['naziv'],
        map['slika'],
        map['pocetak'],
        map['kraj'],
        map['godina'],
        map['brojDana'],
        map['slogan']);
  }
}

File: FITCCRS2.Services/Database/Takmicenje.cs
using System;
using System.Collections.Generic;

namespace FITCCRS2.Services.Database
{
    public partial class Takmicenje
    {
        public Takmicenje()
        {
            Agendums = new HashSet<Agendum>();
            Kategorijas = new HashSet<Kategorija>();
            Tims = new HashSet<Tim>();
        }

        public int TakmicenjeId { get; set; }
        public string? Naziv { get; set; }
        public string? Slogan { get; set; }
        public DateTime? Pocetak { get; set; }
        public DateTime? Kraj { get; set; }
        public int? Godina { get; set; }
        public int? BrojDana { get; set; }
        public string? Slika { get; set; }

        public virtual ICollection<Agendum> Agendums { get; set; }
        public virtual ICollection<Kategorija> Kategorijas { get; set; }
        public virtual ICollection<Tim> Tims { get; set; }
    }
}
